showcase/app/templates/page-components/form/key-value-inputs/frameless/demo-responsiveness-custom-widths.hbs

```diff
@@ -16,23 +16,30 @@
       <H.Generic>
         <Shw::Placeholder @height="36px" @text="generic" />
       </H.Generic>
-      <H.Error>There is a error.</H.Error>
     </:header>
 
     <:row as |R|>
+      {{! @glint-expect-error }}
```

Comment:
I don't know if you want to link to the same Jira ticket as the other similar "expected errors" above or have a different one for this; I'll leave it to you (but add a comment with the Jira ticket please)